packages/github/__tests__/github.proxy.test.ts

```diff
@@ -0,0 +1,109 @@
+import * as http from 'http'
```

Comment:
We now load proxy settings when the module is imported because of the way `defaults` works. This required a new test suite for proxy tests.